src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6697,4 +6697,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_AbstractConversionNotInvolvingContainedType" xml:space="preserve">
     <value>User-defined conversion in an interface must convert to or from a type parameter on the enclosing type constrained to the enclosing type</value>
   </data>
+  <data name="ERR_InterfaceImplementedByUnmanagedCallersOnlyMethod" xml:space="preserve">
+    <value>'UnmanagedCallersOnly' method '{0}' cannot implement interface member '{1}' in type '{2}'</value>
```

Comment:
It looks like `{2}` is included in this diagnostic because it may not be the same as the containing type of `{0}`. Is that correct?

Reply:
> It looks like {2} is included in this diagnostic because it may not be the same as the containing type of {0}. Is that correct?

Correct.